ably/types/tokendetails.py

```diff
@@ -35,9 +42,9 @@ def client_id(self):
     @staticmethod
     def from_dict(obj):
         return TokenDetails(
-            id=obj.get("id"),
+            token=obj.get("token"),
             expires=int(obj.get("expires", 0)),
-            issued_at=int(obj.get("issued_at", 0)),
+            issued=int(obj.get("issued", 0)),
```

Comment:
I am assuming the 0 is a default value. Are we sure we want to default to zero, I would have assumed it would be better to be nil